config/gocdb_schema.xml

```diff
@@ -380,7 +380,7 @@
 	<field>
 	    <fname>EMAIL</fname>
 	    <length>255</length>
-	    <regex>/^([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}$/</regex>
+	    <regex>/^(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6})(;(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}))?$/</regex>
```

Comment:
This only matches one OR two email addresses, if you want to be able to match any number then you probably want:
```suggestion
	    <regex>/^(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6})(;(([0-9a-zA-Z]+[-._])*[0-9a-zA-Z]+?@([-0-9a-zA-Z]+[.])+[a-zA-Z]{2,6}))*$/</regex>
```

Reply:
indeed , thanks a lot !